src/pages-and-resources/edxnotes/Settings.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+
+import AppSettingsModal from '../app-settings-modal/AppSettingsModal';
+import messages from './messages';
+
+function NotesSettings({ intl, onClose }) {
+  return (
+    <AppSettingsModal
+      appId="edxnotes"
+      title={intl.formatMessage(messages.heading)}
+      enableAppHelp={intl.formatMessage(messages.enableNotesHelp)}
+      enableAppLabel={intl.formatMessage(messages.enableNotesLabel)}
+      learnMoreText={intl.formatMessage(messages.enableNotesLink)}
+      onClose={onClose}
+    />
+  );
```

Comment:
like other pages, I cant see settings fetch and update functions here for the notes app.

Reply:
Notes doesn't have any additional settings, just enable/disable which is handled in a different manner.

Reply:
got it. Thanks